src/functionalTest/java/uk/gov/hmcts/divorce/testutil/CaseDataUtil.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.divorce.testutil;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.springframework.util.ResourceUtils.getFile;
+
+public final class CaseDataUtil {
+
+    private static final CaseDataUtil.MapTypeReference MAP_TYPE = new CaseDataUtil.MapTypeReference();
+
+    private CaseDataUtil() {
+    }
+
```

Comment:
Could Lombok annotation be used here?
@NoArgsConstructor(access = AccessLevel.PRIVATE)